heapster.go

```diff
@@ -59,18 +59,17 @@ func main() {
 	os.Exit(0)
 }
 
-func getFlags() []string {
-	flagData := []string{}
-	flag.VisitAll(func(flag *flag.Flag) {
-		flagData = append(flagData, fmt.Sprintf("%s='%v'", flag.Name, flag.Value))
-	})
-	return flagData
-}
-
 func validateFlags() error {
 	if *argPollDuration <= time.Second {
-		return fmt.Errorf("poll duration is invalid '%d'. Set it to a duration greater than a second", *argPollDuration)
+		return fmt.Errorf("Error - poll duration is invalid '%d'. Set it to a duration greater than a second", *argPollDuration)
```

Comment:
nit: Go errors strt with lowercase, also why do we need to append error to all of these?